### ISSUE TITLE
Generated export-from code is incorrect

### ISSUE THREAD

**Issue by ooflorent** [6to5/6to5#406]
The following `export` statement is not properly transformed:

```js
export {A, B} from './foo'
```
It produces:

```js
exports.A = _interopRequire(require("./foo"));
exports.B = _interopRequire(require("./foo"));
```

Instead of:

```js
var _ref = require("./foo");
exports.A = _ref.A;
exports.B = _ref.B;
```

Tested using `6to5@2.6.2`

**Comment by sebmck** [comment]
It's correct. It's essential for module interopability. See #95, #224 and #225 for prior discussion. Unless you're referring to the duplicate `require`s in which case it's another issue and it's not really *incorrect*.

**Comment by ooflorent** [comment]
According to harmony specifications `export {A} from './foo'` should export `A` from `./foo`.
The generated code fails to export `A`.

```js
export {A, B} from './foo'`
```

Expect `ExportBinding`s:

`[[ExportName]]` | `[[ModuleRequest]]` | `[[ImportName]]`
-----------------|---------------------|-----------------
`A`              | `./foo`             | `A`
`B`              | `./foo`             | `B`

Am I wrong?

**Comment by sebmck** [comment]
My apologies, I misread your original issue. This certainly shouldn't be expected behaviour, I'll investigate shortly.

**Comment by ooflorent** [comment]
It is a regression. `6to5@2.4.5` produces:

```js
exports.A = require("./foo").A;
exports.B = require("./foo").B;
```

**Comment by sebmck** [comment]
Really sorry about that. Fixed as of 2.6.3

**Comment by ooflorent** [comment]
:+1:
